examples/src/pages/Migration.tsx

```diff
@@ -375,7 +375,7 @@ export function Migration() {
         cadModels.forEach(m => m.setDefaultNodeAppearance({ visible: !hide }));
       });
 
-      const clippingUi = new ClippingUI(gui.addFolder('Slicing'), planes => viewer.setSlicingPlanes(planes));
+      const clippingUi = new ClippingUI(gui.addFolder('Slicing'), planes => viewer.setClippingPlanes(planes));
```

Comment:
Oh, can you remove `setSlicingPlanes` altogether @Savokr ? It's marked for removal in 3.0

Reply:
Ok